OrbitQt/servicedeploymanager.cpp

```diff
@@ -229,6 +249,9 @@ outcome::result<void> ServiceDeployManager::StartOrbitService() {
 }
 
 outcome::result<void> ServiceDeployManager::StartOrbitServicePrivileged() {
+  // TODO(antonrohr) As soon as std err reading is implemented in
+  // OrbitSshQt::Task, check whether the password was incorrect and return
+  // appropriate error
```

Comment:
I think the better approach is to use sudo just to set the SUID bit and use the exit code to check if that worked. Parsing textual program output is always brittle. But still the output could be displayed to the user.

Reply:
Splitting this up in two tasks might be a good idea. My initial thought was to just use the exit code of sudo (which is 1 when it fails), but since its allowed to try a password again, sudo does not return but waits for more input. Maybe have a task to set the suid bit with a timeout. I am going to update the todo message

Reply:
I tried this. And my version of sudo detects that stdin is already closed and shows the message "sudo: no password was provided" instead of asking a second time for a password.

Reply:
It's like pressing Ctrl+D in an interactive sudo. This gives me the same message.

Reply:
Ah okay, maybe the piece that is missing is sending EOF after the password was sent. I tried it without closing stdin (obv since we want to use OrbitService afterwards) and it asked again for a password